localstack/services/stepfunctions/asl/component/state/state_wait/wait_function/seconds_path.py

```diff
@@ -18,6 +17,10 @@ def __init__(self, path: str):
         self.path: Final[str] = path
 
     def _get_wait_seconds(self, env: Environment) -> int:
-        input_expr = parse(self.path)
-        seconds = input_expr.find(env.inp)
+        inp = env.stack[-1]
```

Comment:
nice improvement to get the `input` from the stack rather than parsing and finding it 👍